lib/imageCacheHoc.js

```diff
@@ -75,24 +97,25 @@ export default function imageCacheHoc(Image, options = {}) {
      *
      * @returns {Promise} promise that resolves to an object that contains the flush results.
      */
-    static async flush() {
-
-      const fileSystem = FileSystemFactory();
-      const results = await Promise.all([fileSystem.unlink('permanent'), fileSystem.unlink('cache')]);
+    static async flush(fileDirName = null) {
```

Comment:
Added `fileNameDir` here for parity, and to make sure we can explicitly flush a specific cache directory.